docs/using-hive.md

```diff
@@ -104,15 +104,41 @@ spec:
     name: mycluster-ssh-key
 ```
 
-The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the cluster defined by your current kubeconfig.
+The hiveutil CLI (see `make hiveutil`) offers a create-cluster command for generating a cluster deployment and submitting it to the Hive cluster using your current kubeconfig.
+
+To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+
+By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--hive-image` can be specified to use a specific Hive image to run the install, and `--release-image` can be specified to control which OpenShift release image to install in the cluster.
+
+#### Create Cluster on AWS
+
+Credentials will be read from your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. Alternatively you can specify an AWS credentials file with --creds-file.
 
 ```bash
-bin/hiveutil create-cluster --base-domain=mydomain.example.com mycluster
+bin/hiveutil create-cluster --base-domain=mydomain.example.com --cloud=aws mycluster
 ```
 
-To view what create-cluster generates, *without* submitting it to the API server, add `-o yaml` to the above command. If you need to make any changes not supported by create-cluster options, the output can be saved, edited, and then submitted with `oc apply`.
+#### Create Cluster on Azure
 
-By default this command assumes the latest Hive master CI build, and the latest OpenShift stable release. `--hive-image` can be specified to use a specific Hive image to run the install, and `--release-image` can be specified to control which OpenShift release image to install in the cluster.
+Credentials will be read from ~/.azure/osServicePrincipal.json typically created via the `az login` command.
+
+```bash
+bin/hiveutil create-cluster --base-domain=azure.mydomain.example.com --cloud=azure mycluster
```

Comment:
We should include the `--azure-base-domain-resource-group-name` flag in the example, as a user is likely going to need to use it.